app/build.gradle

```diff
@@ -186,6 +186,7 @@ android {
             buildConfigField "String", "WIKIMEDIA_API_HOST", "\"https://commons.wikimedia.org/w/api.php\""
             buildConfigField "String", "WIKIDATA_API_HOST", "\"https://www.wikidata.org/w/api.php\""
             buildConfigField "String", "WIKIMEDIA_FORGE_API_HOST", "\"https://tools.wmflabs.org/\""
+            buildConfigField "String", "WIKIMEDIA_CAMPAIGNS_URL", "\"https://raw.githubusercontent.com/commons-app/campaigns/master/prod/campaigns.json\""
```

Comment:
Now that we are updating the url, the original json needs to be added at this location as well. Presently it is not. You might need to raise a pr for the same  at https://github.com/commons-app/campaigns